heka/files/lua/encoders/status_nagios.lua

```diff
@@ -19,7 +19,9 @@ local consts = require 'gse_constants'
 local lma = require 'lma_utils'
 local interp = require "msg_interpolate"
 
-local host = read_config('nagios_host')
+local default_host = read_config('default_nagios_host')
+local host_dimension_key = read_config('nagios_host_dimension_key')
```

Comment:
nope, because the same encoder is used for AFD .. which use Fields[hostname] or Hostname as nagios_host